### DISCUSSION TITLE
Adding another text along with street name

### DISCUSSION THREAD

**Discussion by anithaswain-tomtomexternal** [openstreetmap/iD#10132] [General]
Im trying to add a number on road element at start point but when there is a street name on road element, the number starts displaying from the start point of street name but not on start point of path.Please suggest how to resolve this styling issue

**Comment by tyrasd** [comment]
It's not quite clear to me what you're stuck on. Can you please rephrase your question @anithaswain-tomtomexternal ?

**Comment by anithaswain-tomtomexternal** [comment]
i have resolved it. Thanks for your  response but next time pls respond quickly
